OrbitVulkanLayerClientGgp/LayerLogic.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "LayerLogic.h"
+
+#include <math.h>
+#include <string.h>
+#include <unistd.h>
```

Comment:
I would expect clang format to protest here:

````
#include "LayerLogic.h"

#include <unistd.h>

#include <math.h>
#include <string.h>
````
would be my expectation. Please ignore if clangformat is fine with this.

Reply:
clangformat seems to be fine with this. So please ignore ...